Cataclysm/Rogue.lua

```diff
@@ -1000,13 +1038,11 @@ spec:RegisterAbilities( {
         cast = 0,
         cooldown = 10,
         gcd = "totem",
-
-        spend = function() return glyph.gouge.enabled and 30 or 45 end,
+        requiresFacing = function() return not glyph.gouge.enabled end,
```

Comment:
Is this used somewhere meaningful?

Reply:
it will be standard as a pic for glyph when without rogues need to be behind when glyphed it dosnt matter, but over all i think everyone will have i glyphed